tests/console/system_prepare.pm

```diff
@@ -68,7 +68,7 @@ sub run {
         }
     }
 
-    my $nss_systemd = script_run('if [ -f /etc/nsswitch.conf ]; then grep passwd.*systemd /etc/nsswitch.conf; fi');
+    my $nss_systemd = script_run('if [ -f /usr/etc/nsswitch.conf -a -f /etc/nsswitch.conf ]; then grep passwd.*systemd /etc/nsswitch.conf; fi');
```

Comment:
I would rather create a symlink. I know in our usrmerge files in `/etc` are local files which overwrite `/usr/etc` but IMHO there still a lot of software which just expect `/etc/nsswitch.conf` and it does not use `/usr/etc/nsswitch.conf`.